portal/portal-api/api/src/java/org/sakaiproject/portal/beans/bullhornhandlers/GradeAssignmentBullhornHandler.java

```diff
@@ -70,7 +70,13 @@
                 List<BullhornData> bhEvents = new ArrayList<>();
                 submission.getSubmitters().forEach(to -> {
                     try {
-                        bhEvents.add(new BullhornData(from, to.getSubmitter(), siteId, title, assignmentService.getDeepLink(siteId, assignment.getId(), to.getSubmitter())));
+                        String url = assignmentService.getDeepLink(siteId, assignment.getId(), to.getSubmitter());
+                        // TODO fix Assignment so that the submitter list doesn't contain users that have dropped the course (SAK-42721)
```

Comment:
This is an important point about if a user is removed from a site do you scrub them from any content in the site?
- remove their assignments
- remove their resources 
- remove forum posts

Sakai has always benefitted from not removing a users content as once their added back all their content reappears.

Imagine if a user was accidentally removed from a roster via SIS and then Sakai removed all that users content, and then once SIS was fixed the user found all of their content removed!!! Not a fun spot to be in...